dashboard/src/api/api.js

```diff
@@ -0,0 +1,8 @@
+// apiGet performs a GET request on the API with given local URL.
+// The result is decoded from JSON and returned.
+export async function apiGet(localURL: string): Promise<Any> {
```

Comment:
Here you're using TypeScript syntax but the files are JS files.

If you're going to use Create React App you probably want to use it with the TypeScript react-scripts:
https://github.com/Microsoft/TypeScript-React-Starter#create-our-new-project

Reply:
BTW if you do use this, make sure to simplify the `extends` list in the tslint.json it generates:
```
"extends": ["tslint-config-prettier"],
```
The defaults are overly pedantic and tend to get in the way.